ibm/service/cis/resource_ibm_cis_firewall_rules.go

```diff
@@ -238,6 +238,21 @@ func ResourceIBMCISFirewallrulesDelete(context context.Context, d *schema.Resour
 		return diag.FromErr(fmt.Errorf("[ERROR] Error deleting the  custom resolver %s:%s", err, response))
 	}
 
+	if id, ok := d.GetOk(cisFilterID); ok {
```

Comment:
Can we use same filterID forother rules. If we delete the filterID will not be other firewall rules not effected?
If user create a filter using Terraform is the destroy not handled to remove filter

Reply:
No we can not use same filter for other rules. It throws an error on creating a new rule with an already mapped filterID.

Reply:
Yes, destroy handles filter deletion but it does not cover old filters.